helm/cilium/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: cilium
 displayName: Cilium
 home: https://cilium.io/
-version: 0.18.0
-appVersion: 1.14.3
+version: 1.14.4
```

Comment:
Was this a conscious decision to go from 0.18.0 to 1.14.4?

Reply:
Ping @giantswarm/team-cabbage

Reply:
It was. Since we switched to vendir to handle upstream chart, we get that value from upstream. Nevertheless, when building the chart, `abs` overrides the value with our release version so we should be good